### PR TITLE
TECH-141: Create test configuration for "/authorization/send-otp" endpoint

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
Create test configuration for "/authorization/send-otp" endpoint.

Create test configuration for /authorization/oauth-details endpoint from https://github.com/GovStackWorkingGroup/bb-identity/pull/6.
Identify the critical APIs for Authentication, write the test plan and Cucumber tests, as well as supporting code to run those Cucumber tests.

TICKET: https://govstack-global.atlassian.net/browse/TECH-141

### CODE REVIEW

**Review by mgrochalskisoldevelo — CHANGES_REQUESTED** [review]
There is something I'd like to confirm.

image

The description in response tells us ```null if failed to deliver OTP```
Does it apply to response or messageCode ? @KarolinaKopacz 
To consider is that one of those can come as null

Also added 1 suggestion in the comment.

File: test/features/support/ui_authorization_send_otp.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      },
+    };
+  }
+
+  specUIAuthSendOTP.expectStatus(200).expectJsonSchema(responseSchema);
+  await specUIAuthSendOTP.toss();
+};
+
+Before(() => {
+  specUIAuthSendOTP = pactum.spec();
```
[the remark this answers is not in the document]
Comment (KarolinaKopacz) [reply test/features/support/ui_authorization_send_otp.js:80]: Thank you!

File: test/features/support/ui_authorization_send_otp.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  specUIAuthSendOTP.expectResponseTime(1500);
```
Comment (mgrochalskisoldevelo) [inline test/features/support/ui_authorization_send_otp.js:82]: Please change it to 15000. It's an amount in milliseconds

**Review by mgrochalskisoldevelo — CHANGES_REQUESTED** [review]
I left a comment with small issue. Except that LGTM

**Review by mgrochalskisoldevelo — APPROVED** [review]
LGTM

**Comment by KarolinaKopacz** [comment]
I'm going to close this PR because endpoint no longer exist in the bb-identity API spec.

**Pull request closed without merging**